src/main/java/org/fcrepo/storage/ocfl/OcflObjectSession.java

```diff
@@ -150,6 +150,14 @@
      */
     void versionMessage(final String message);
 
+    /**
+     * Invalidate the cache for a Fedora resource's headers
+     *
+     * @param resourceId the Fedora resource id to read
+     * @param versionNumber the version to read, or null for HEAD
```

Comment:
I don't think `versionNumber` can be `null` here. I think it always needs to be an actual version.